packages/application-shell/src/components/application-shell/application-shell.spec.js

```diff
@@ -515,14 +514,15 @@ describe('when user is not authenticated', () => {
     const rendered = renderApp(null, { route: '/foo' });
     const queryParams = encode({
       reason: LOGOUT_REASONS.UNAUTHORIZED,
-      redirectTo: `${window.location.origin}/foo`,
```

Comment:
@emmenko this is the only test I am not too sure about. I am not sure why it should be `/foo`. Cause the window.location.origin isn't on `/foo`. It's the window.history and react-router which is.

It's also another type without an actual origin.

<img width="507" alt="CleanShot 2020-03-16 at 19 27 31@2x" src="https://user-images.githubusercontent.com/1877073/76789432-1b234e80-67bd-11ea-9f8b-7132a761bd02.png">

Reply:
I'm not sure I get what you mean. The `/foo` route is what it needs to redirect to, so it needs to be in the `redirectTo` param.

Reply:
Ah true my bad. Will check again tomorrow.